thefuck/rules/docker_image_being_used_by_container.py

```diff
@@ -0,0 +1,13 @@
+from thefuck.utils import for_app
+
+
+@for_app('docker')
+def match(command):
+    return ('docker' in command.script
```

Comment:
`@for_app('docker')` already ensures that the `command.script` starts with `docker`.